packages/web/src/components/typography/Text.tsx

```diff
@@ -0,0 +1,87 @@
+// import cn from 'classnames'
+
+import { CSSProperties, ReactNode } from 'react'
+
+import { getCurrentThemeColors, ThemeColor } from 'utils/theme/theme'
+
+import { variantInfoMap, variantStrengthMap } from './constants'
+import {
+  TextSizeInfo,
+  TextStrengthInfo,
+  TextVariant,
+  TextSize,
+  TextStrength,
+  FontWeight
+} from './types'
+
+export const fontWeightMap: Record<FontWeight, number> = {
```

Comment:
would it make sense to move these constants into the types file with some of the other definitions?